src/vault.py

```diff
@@ -362,7 +362,11 @@ def fuzz(args):
 # Google dork
 
 def dork(args):
-    if args.dork:
+    if not args.dork:
```

Comment:
It should be `if not args.url`, also put the import in try-catch block, take example from any other similar function.

Reply:
Okay